src/pages/SampleListPage.tsx

```diff
@@ -53,16 +67,109 @@ export const SampleListPage = () => {
     }));
   }, [data]);
 
+  function handleDownloads(type: string, infoText: string, dataFormat: string = '') {
+    let fileURL =
+      dataFormat === 'csv'
+        ? getLinkTo(type, accessions, undefined, true, 'csv')
+        : getLinkTo(type, accessions, undefined, true);
+    let tempLink = document.createElement('a');
+    tempLink.href = fileURL;
+    tempLink.click();
+  }
+
+  useEffect(() => {
+    if (selectionModel.length > 0 && rows) {
+      let selected = selectionModel.map((i: number) => {
+        return rows[i].accession;
+      });
+      setAccessions({ identifier: { accession: selected } });
+    }
+  }, [selectionModel, rows]);
+
   if (!rows) {
     return <>Loading...</>;
   }
 
   return (
     <>
-      <PageHeaderWithReturn title='Selected samples' to={`../explore${searchString}`} />
+      {selectionModel.length > 0 && (
+        <div className='m-8 flex flex-row flex-wrap'>
+          <div className='mx-4 my-1' style={{ zIndex: 10 }}>
```

Comment:
Would it be possible/better to reuse the `TopButtons` component? ([code](https://github.com/cevo-public/mpox-spectrum-website/blob/673049bfc8ff9dccea563817b184685ce214a304/src/components/TopButtons.tsx))